includes/core/server/request/class-list-transactions.php

```diff
@@ -0,0 +1,80 @@
+<?php
+/**
+ * Class file for WCPay\Core\Server\Request\List_Transactions.
+ *
+ * @package WooCommerce Payments
+ */
+
+namespace WCPay\Core\Server\Request;
+
+use WC_Payments_API_Client;
+use WC_Payments_DB;
+/**
+ * Request class for listing transactions.
+ */
+class List_Transactions extends Paginated {
+
+	use Order_Info;
+
+	const DEFAULTS = [
+		'page'       => 0,
+		'pagesize'   => 25,
+		'sort'       => 'date',
+		'direction'  => 'desc',
```

Comment:
Defaults are inherited from parents, at least `page` and `pagesize` do not to be re-defined here.

Reply:
This is fixed.